schemas/mysql.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+echo "CREATE DATABASE IF NOT EXISTS homes_for_heroes;" >  /mainschema.sql
+echo "USE homes_for_heroes;" >>  /mainschema.sql
+echo "ALTER USER root@localhost IDENTIFIED WITH caching_sha2_password BY '$MYSQL_ROOT_PASSWORD';">>  /mainschema.sql
+echo "CREATE USER IF NOT EXISTS '$MYSQL_USER'@'%' IDENTIFIED WITH caching_sha2_password BY '$MYSQL_PASSWORD';" >> /mainschema.sql
```

Comment:
Why is the password injected into the schema SQL? The docker container already should be initialized with the env credentials.